backend/api/EventHandlers/MissionScheduler.cs

```diff
@@ -13,10 +13,10 @@ public class MissionScheduler : BackgroundService
         private readonly int _timeDelay;
         private readonly IServiceScopeFactory _scopeFactory;
 
-        private IList<Mission> MissionQueue =>
+        private IList<MissionRun> MissionQueue =>
             MissionService
```

Comment:
```suggestion
            MissionRunService
```

I would expect this line to break the code currently as you have renamed the `MissionService` to `MissionRunService`